app/dbmodels.py

```diff
@@ -42,9 +39,58 @@ class Meta(BaseMeta):
     settings: pydantic.Json = ormar.JSON()
 
 
+class Conversations(ormar.Model):
+    class Meta(BaseMeta):
+        tablename = "conversations"
+    id: int = ormar.Integer(primary_key=True)
+    type: str = ormar.String(max_length=100)
+    settings: pydantic.Json = ormar.JSON()
+    owner_id: int = ormar.Integer()
+    created_at: int = ormar.Integer()
+    updated_at: int = ormar.Integer()
 
-engine = sqlalchemy.create_engine(config.DATABASE_URL)
-metadata.create_all(engine)
+
+class ConversationUsers(ormar.Model):
+    class Meta(BaseMeta):
+        tablename = "conversation_users"
+    id: int = ormar.Integer(primary_key=True)
+    conversation_id: int = ormar.Integer()
```

Comment:
```suggestion
    conversation_id: Conversation = ormar.ForeignKey(Conversation)
```
Опять же, внешний ключ. Далее по коду то же самое, следует исправить

Reply:
Не забудь в аннотации типов так же указывать нужный класс (после двоеточия, перед знаком "=")